api/app/config/detekt.yml

```diff
@@ -1,3 +1,6 @@
+build:
```

Comment:
can we get rid of this and just fix the issues please? I fixed them in a previous commit, have they crept back in? or, if we don't care about lexicographic ordering of imports, just disable that rule altogether

Reply:
This is actually an old PR. I just updated with master.